src/ports/items/component.ts

```diff
@@ -25,7 +26,11 @@ export function createItemsComponent(
       items: ItemFragment[]
     }>(query)
     const items = fragments.map((fragment) =>
-      fromItemFragment(fragment, network, chainId)
+      fromItemFragment(
+        fragment,
+        network as Network.ETHEREUM | Network.MATIC,
```

Comment:
is this necessary? fromItemFragment is receiving AssetsNetwork too

Reply:
This `network` variable comes from the `ItemFilters`, which is still using the `Rarity` type. As changing it might impact multiple projects, I chose casting it here, but I made sure that we were getting one or the other when receiving the query parameters.